quill-core/src/main/scala/io/getquill/ast/Ast.scala

```diff
@@ -104,4 +106,11 @@ case class Assignment(input: Ident, property: String, value: Ast)
 
 //************************************************************
 
+sealed trait SchemaDefinition extends Ast
```

Comment:
It would be wonderful, do you have any idea on how to do this? Right now, I am using a rather complex transformation to flatten the `SchemaDefinition` on top of `Entity`.